ONSdigital/dp-mongodb
# Create mongo health checker

### What

Moved [go-ns mongo](https://github.com/ONSdigital/go-ns/tree/master/mongo) package into new mongodb library/repo.

Also created new health checker abiding by [specification](https://github.com/ONSdigital/dp/blob/master/standards/HEALTH_CHECK_SPECIFICATION.md), includes unit tests

### How to review

* Check settings are correct
* Check new repo has all relevant components, e.g. documentation, license etc.
* Health checker abides with healthcheck documentation, see link above
* Unit tests pass

### Who can review

Anyone
